solutions/devsprint-bruno-ramos-3/DeliveryApp/Screens/AddressSearch/AddressCellView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  AddressCellView.swift
+//  DeliveryApp
+//
+//  Created by Pedro Henrique Léda on 15/02/22.
+//
+
+import Foundation
+import UIKit
+
+class AddressCellView: UITableViewCell {
+    var stackView: UIStackView = {
```

Comment:
Como não iremos precisar utilizar essa propriedade fora da classe, prefira manter como privada
Também sugiro ser `let` ao invés de `var` dado que esse valor não deve ser alterado
```suggestion
    private let stackView: UIStackView = {
```